lib/renderer/src/renderer/scene/rust/RustRenderer.cpp

```diff
@@ -0,0 +1,192 @@
+#include <iostream>
+

```

Comment:
would be nice to have a similar protection mechanism here that checks the API version like we do in rv_ffi

Reply:
Sure, I'll add it definitely